l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -617,6 +617,50 @@ def setDettaglioLinee(self, invoice, body):
                 line_no, line, body, price_precision, uom_precision)
             line_no += 1
 
+        generic_mngt_line = False
+        generic_mngt_lines = invoice.related_mngt_data_ids.filtered(
+            lambda x: not x.lineRef)
+        if generic_mngt_lines:
+            generic_mngt_line = generic_mngt_lines[0]
+        for DettaglioLinea in body.DatiBeniServizi.DettaglioLinee:
+            # get related mgnt_line if exists
+            mngt_lines = filter(
+                lambda x: x.lineRef == DettaglioLinea.NumeroLinea,
+                invoice.related_mngt_data_ids)
+            if mngt_lines:
+                for mngt_line in mngt_lines:
+                    dati_gestionali = AltriDatiGestionaliType()
+                    if mngt_line.name:
+                        dati_gestionali.TipoDato = mngt_line.name
+                    if mngt_line.text_ref:
+                        dati_gestionali.RiferimentoTesto = mngt_line.text_ref
+                    if mngt_line.number_ref:
+                        dati_gestionali.RiferimentoNumero = '%.2f' % \
+                            mngt_line.number_ref
+                    if mngt_line.date_ref:
+                        dati_gestionali.RiferimentoData = mngt_line.date_ref
+                    DettaglioLinea.AltriDatiGestionali.append(
+                        dati_gestionali
+                    )
+            else:
+                if generic_mngt_line:
+                    dati_gestionali = AltriDatiGestionaliType()
+                    # if fatturapa line is not referred, and exist a
+                    # generic_mngt_line, add this generic mngt data to line
```

Comment:
Se in una fattura con una riga inserisco 10 `FatturaPA Mngt Data Type` di cui uno solo ha il `lineRef` corretto, solo quello verrà inserito nella fattura elettronica.

Va chiarito all'utente (e ai dev) il legame tra questo nuovo oggetto e la riga fattura, e questo `generic_mngt_line` confonde un po' le idee

Reply:
Le righe di Related Mngt Data vengono associate alle righe fattura tramite la lineRef. e il numero della riga fattura (sequence).
Le righe con lineRef = 0 vengono associate a tutte le righe fattura per cui non ho specificato una Related Mngt Data.
Possono essere specificate più righe di Related Mngt Data per ogni riga fattura.